clayui.com/src/components/Editor/index.js

```diff
@@ -12,7 +12,7 @@ import React, {useContext, useState} from 'react';
 import {LiveEditor, LiveError, LivePreview, LiveProvider} from 'react-live';
 
 import theme from '../../utils/react-live-theme';
-import useStickyState from '../Hooks/useStickyState';
+import useStateWithLocalStorage from '../Hooks/useStateWithLocalStorage';
```

Comment:
@kresimir-coko I renamed this just so we can be explicit as possible

Reply:
`<devils-advocate>`At the same time, you've leaked the implementation details into the name, arguably making it a leakier abstraction now, and the new name is less descriptive of the purpose and behavior of the hook (ie. before, the word "sticky" conveyed that the state somehow sticks around beyond the normal lifecycle of the component; after, the term "with local storage" conveys that local storage is used but it doesn't really say how/why).`</devils-advocate>`

I have no problem with the change, but I just wanted to point out that "explicit" is not always necessarily better than "abstract".

Reply:
An example I should have mentioned was, what if, for some reason, we decided to switch to `sessionStorage` from `localStorage` in the future? Would you rename the hook to `useStateWithSessionStorage`?